src/index.html

```diff
@@ -11,7 +11,149 @@
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="label">
+          How old are You?
+          <input 
+            type="number" 
+            name="age" 
+            min="1" 
+            max="100" 
+            value="12" 
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="label">
+          Full date of birth:
+          <input 
+            type="date" 
+            name="date" 
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="label last_label">
+          I accept the term of the agreement
+          <input type="checkbox" name="checkbox">
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label class="label">
+          E-mail:
+          <input 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com" 
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="label last_label">
+            Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="5" 
+            maxlength="32" 
+            autocomplete="off"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label>
+          Do you love cats?
+          <input type="radio" name="sex">
+          Yes
+        </label>
+        <label>
+          <input type="radio" name="sex">
+          No
+        </label>
+
+        <label class="label_color">
+          What is your favorite color?
+          <input type="color" name="input">
+        </label>
+
+        <label class="label">
+          What time do you go to bed?
+          <input 
+            type="time" 
+            name="time" 
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="label">
+          What are your favorite brand of cars?
+          <select name="car" multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="last_label">
+          How do you rate our work?
+          <input type="range" name="range">
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info:</legend>
+        
+        <label class="label">
+          Comments:
+          <textarea 
+            name="textarea" 
+            cols="20" 
+            rows="2" 
+            autocomplete="off"
+          ></textarea>
+        </label>
+
+        <label class="last_label">
+          Would you recommend us?
+          <select name="recommend">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+        
+      </fieldset>
```

Comment:
there shouldn’t be empty line between child and parent element
```suggestion
        </label>
      </fieldset>
```